test/test_datasets_wildfire_split.py

```diff
@@ -93,6 +93,15 @@ def test_inconsistent_ratios_raise_exception(self):
         with self.assertRaises(ValueError):
             WildFireSplitter(ratios)
 
+    def test_splitting_with_test_to_zero(self):
+        ratios = {'train': 0.81, 'val': 0.19, 'test': 0}
+
+        splitter = WildFireSplitter(ratios, seed=42)
+        splitter.fit(self.wildfire)
+
+        for (set_, ratio_) in splitter.ratios_.items():
+            self.assertAlmostEqual(ratio_, ratios[set_], places=2)
```

Comment:
Is it required to have so few decimals in `assertAlmostEqual`?
If the precision can be determined using the size of the dataset to be more accurate, that would be even better!

Reply:
These are estimated ratios and 2 decimals is enough here because most important constraint is to group frames referring to same fire.  One may improve assigning algorithm, but biasing in order to be better at respecting ratios size is not a good idea IMO. This is not the kind of biases that would improve predictability.